spark/dl/src/main/scala/com/intel/analytics/bigdl/example/languagemodel/PTBModel.scala

```diff
@@ -49,7 +49,7 @@ object PTBModel {
                input: ModuleNode[Float]): ModuleNode[Float] = {
     if (depth == numLayers) {
       Recurrent[Float]()
-        .add(LSTM[Float](inputSize, hiddenSize, 0, null, null, null))
+        .add(LSTM[Float](inputSize, hiddenSize, 0, null, null, null, null, null))
```

Comment:
I think two new added parameters have default value, we don't need change this file?

Reply:
Yeah, no need to change this file. My bad :(